### DISCUSSION TITLE
Welcome to Flavour-Fusion Discussions!

### DISCUSSION THREAD

**Discussion by c4coderandcreator** [c4coderandcreator/Flavour-Fusion#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by c4coderandcreator** [comment]
@Shybymh hey the order now section is way to large reduce it height and width  !! Also maka the close button look appealing !! and Try to make it responsive !! Though I am listing these in issues !!

**Reply by Shybymh** [reply]
I see that these issues are assigned now.

**Comment by c4coderandcreator** [comment]
> I see that these issues are assigned now.

I understood that you are paying efforts to this project to get your contribution counted , and I really appreciate that !! Thanks @Shybymh
